ui/pages/setup_secure.tsx

```diff
@@ -81,11 +81,11 @@ const SetupSecure: NextPage = () => {
         })
         .then(({ data }) => {
           if ("redirect_to" in data) {
-            window.location.href = data.redirect_to as string;
+            window.location.href = `./setup_complete?flow=${data.id}`;
             return;
           }
```

Comment:
This is not needed, we can remove this case. The backend should always return the flow

Reply:
removed this bit.